Gopkg.lock

```diff
@@ -2,191 +2,248 @@
 
 
 [[projects]]
+  digest = "1:167b6f65a6656de568092189ae791253939f076df60231fdd64588ac703892a1"
```

Comment:
I am thinking the dep file format change should be in its own PR, to avoid it being hidden through this PR, and to make sure ppl realize it's been updated.

Reply:
Agreed. I'll raise a PR for that later today...

Reply:
Right - #574 raised. I'll add dnm to this PR so when #574 is merged I can remove my first commit from this PR...